### PR TITLE
Debate logo v2 devas 2051

### PR DESCRIPTION
**Pull request by felixsaintgenis**
- add the logo input in personalizeInterface form 

I added a new query to navbar.jsx,  not sure it's a good idea.

### CODE REVIEW

File: assembl/static2/js/app/components/administration/discussion/personalizeInterface/load.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ export function postLoadFormat(data: DiscussionPreferencesQueryQuery): Personali
   const { discussionPreferences } = data;
   return {
     title: discussionPreferences && discussionPreferences.tabTitle,
-    favicon: (discussionPreferences && discussionPreferences.favicon) || null
+    favicon: (discussionPreferences && discussionPreferences.favicon) || null,
```
Comment (MilanBarande) [inline assembl/static2/js/app/components/administration/discussion/personalizeInterface/load.js]: I would destructure discussionPreferences above the return ```const { tabTitle, favicon, logo) = data.discussionPreferences;```
and below write each property like this ```title: tabTitle || null;```
current syntax is a bit odd (due to linter)

File: assembl/static2/tests/unit/components/navbar/__snapshots__/navbar.spec.jsx.snap [hunk 2]
```diff
@@ -38,6 +38,7 @@ exports[`AssemblNavbar component should render a BurgerNavbar and a hidden FlatN
           ]
         }
         logoLink="http://www.homesweethome.org"
+        logoSrc={null}
```
Comment (MilanBarande) [inline assembl/static2/tests/unit/components/navbar/__snapshots__/navbar.spec.jsx.snap]: Add a test with a logo

File: assembl/static2/js/app/components/administration/discussion/personalizeInterface/load.js (hunk 1 above)
Comment (felixsaintgenis) [reply assembl/static2/js/app/components/administration/discussion/personalizeInterface/load.js]: #oui

File: assembl/static2/tests/unit/components/navbar/__snapshots__/navbar.spec.jsx.snap (hunk 2 above)
Comment (felixsaintgenis) [reply assembl/static2/tests/unit/components/navbar/__snapshots__/navbar.spec.jsx.snap]: sure

File: assembl/static2/js/app/components/administration/discussion/personalizeInterface/load.js (hunk 1 above)
Comment (felixsaintgenis) [reply assembl/static2/js/app/components/administration/discussion/personalizeInterface/load.js]: ça génère une erreur flow, je garde l'ancienne syntaxe

File: assembl/graphql/docstrings.py [hunk 3]
```diff
@@ -161,6 +162,7 @@ class UpdateDiscussionPreferences:
     languages = """The list of languages in ISO 639-1 locale code that the debate should support."""
     tab_title = DiscussionPreferences.tab_title
     favicon = DiscussionPreferences.favicon
+    logog = DiscussionPreferences.logo
```
Comment (vincentfretin) [inline assembl/graphql/docstrings.py]: logog => logo

File: assembl/graphql/discussion.py [hunk 4]
```diff
@@ -564,6 +572,7 @@ class Input:
         tab_title = graphene.String(description=docs.UpdateDiscussionPreferences.tab_title)
         # this is the identifier of the part in a multipart POST
         favicon = graphene.String(description=docs.UpdateDiscussionPreferences.favicon)
+        logo = graphene.String(description=docs.UpdateDiscussionPreferences.favicon)
```
Comment (vincentfretin) [inline assembl/graphql/discussion.py]: favicon => logo

**Pull request merged**